src/Analysis/Ast/Test/GenericsTests.cs

```diff
@@ -1075,6 +1075,109 @@ class Box(Generic[_T]):
                 .Which.Should().HaveType(BuiltinTypeId.Str);
         }
 
+        [TestMethod, Priority(0)]
+        public async Task GenericForwardRef() {
+            const string code = @"
+from typing import List, Dict
+class A:
+    def test(self) -> int:
+        pass
+class B:
+    def test(self) -> int:
+        pass
+l = List['A']
+d = Dict['A', 'B']
```

Comment:
Does it work if types are declared later in the file - since it is supposed to be 'forward' reference?

Reply:
Ok I'll check